### ISSUE TITLE
Make this repo private

### ISSUE THREAD

**Issue by maddydeason** [WomenWhoCodeBoston/leaders-minutes#2]
I feel like this would need to be a private repo? @bryanaknight you said you know how to set that up?

**Comment by bryanaknight** [comment]
Yeah private repos are $$ so gotta figure out how to apply a staff coupon to this org I think

**Comment by maddydeason** [comment]
Ah ok, well if we figure it out we can do it. For now we can keep it to the slack. @RachelMinto

**Comment by bryanaknight** [comment]
Im pretty sure I can make it happen! Just gotta do a little digging.
